internal/drand-cli/dkg_cli.go

```diff
@@ -257,15 +257,7 @@ func parseInitialProposal(c *cli.Context) (*drand.FirstProposalOptions, error) {
 	period := c.Duration(periodFlag.Name)
 	timeout := time.Now().Add(c.Duration(dkgTimeoutFlag.Name))
 
-	// if a custom time hasn't been added, we set a one based on the period
-	// to make the tests a little more predictable
-	var genesisTime time.Time
-	if c.IsSet(transitionTimeFlag.Name) {
-		genesisTime = time.Now().Add(c.Duration(transitionTimeFlag.Name))
-	} else {
-		defaultRoundsToWait := period * 3
-		genesisTime = time.Now().Add(defaultRoundsToWait)
-	}
+	genesisTime := time.Now().Add(c.Duration(transitionTimeFlag.Name))
```

Comment:
I'm not super convinced by this. Why adding it to now instead of setting it to the provided value? Wouldn't it be much nicer to be able to specify a given time for Genesis?